R/distances.R

```diff
@@ -455,16 +455,15 @@ distancesMatrix <- function(t.layer, starters = NULL, targets = NULL,
     if(!is.null(starters) | !is.null(targets)) {
       # distancesMatrix() crashes if actel = TRUE and starters/targets were 
       # supplied but do not contain a column called "Standard.name"
-      if(!is.null(starters) & !any(grepl(c("Standard.name"), 
-                                         colnames(starters)))) {
-        event("'starters' is missing column called 'Standard.name'",
-              type = "stop")
+      if(!is.null(starters) && !("Standard.name" %in% colnames(starters))) {
+        stop("'starters' is missing column called 'Standard.name'",
+             call. = FALSE)
       }
       
       if(!is.null(targets) & !any(grepl(c("Standard.name"), 
                                          colnames(targets)))) {
```

Comment:
Just realized we should change this one too

```suggestion
      if(!is.null(targets) && !("Standard.name" %in% colnames(targets))) {
```